tests/unit/conditions/test_return_value.py

```diff
@@ -23,6 +23,47 @@
 from nucypher.policy.conditions.lingo import ReturnValueTest
 
 
+def test_return_value_key():
+    test = ReturnValueTest(comparator=">", value="0", key="james")
+    assert test.eval({"james": 1})
+    assert not test.eval({"james": -1})
+
+    with pytest.raises(KeyError):
+        test.eval({"bond": 1})
+
+    test = ReturnValueTest(comparator=">", value="0", key=4)
+    assert test.eval({4: 1})
+    assert not test.eval({4: -1})
+
+    with pytest.raises(KeyError):
+        test.eval({5: 1})
+
+
+def test_return_value_index():
+    test = ReturnValueTest(comparator=">", value="0", key=0)
+    assert test.eval([1])
+    assert not test.eval([-1])
+
+    test = ReturnValueTest(comparator="==", value='"james"', key=3)
+    assert test.eval([0, 1, 2, '"james"'])
```

Comment:
Why use triple quotation here?

Reply:
it's not triple, it's a single outside of a double, similar to how the other tests are done

Reply:
The line that requires this is `lingo.py:116` if you run that line on a regular string you get:
```
$python
>>> v = "james"
>>> import ast
>>> ast.literal_eval(str(v))
```

Reply:
There is even a pre-existing test for it:
```python
def test_return_value_test_string():
    # string values must be quoted
    with pytest.raises(ReturnValueTest.InvalidExpression):
        _test = ReturnValueTest(comparator="==", value="foo")
```

Reply:
Got it. I'm glad we reviewed it since it's not obvious why its needed.